### PR TITLE
[iam] Correct server address

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

File: src/iam/server/iamserver.cpp [hunk 1]
```diff
@@ -76,6 +76,19 @@ void ConvertToProto(const Array<StaticString<cSubjectIDLen>>& src, iamanager::v4
     }
 }
 
+static const std::string CorrectAddress(const std::string& addr)
+{
+    if (addr.empty()) {
+        throw std::runtime_error("bad address");
```
Comment (mlohvynenko) [inline src/iam/server/iamserver.cpp:82]: Will this exception be handled in Init?

Comment (al1img) [inline src/iam/server/iamserver.cpp:82]: throw Aos exception

**Review by MykolaSolyanko — APPROVED** [review]

File: src/iam/server/iamserver.cpp (hunk 1 above)
Comment (mykola-kobets-epam) [reply src/iam/server/iamserver.cpp:82]: Init method handles std::exception, which runtime_error is a subclass of.

Comment (mykola-kobets-epam) [reply src/iam/server/iamserver.cpp]: changed

**Review by al1img — APPROVED** [review]

**Pull request merged**